### PR TITLE
Use implementation-neutral language about the catalog in workflow process labels

### PR DESCRIPTION
**Pull request by mjgiarlo**
## Why was this change made? 🤔

This is less confusing and less brittle.

## How was this change tested? 🤨

- [x] CI
- [x] https://argo-qa.stanford.edu/report/workflow_grid

### CODE REVIEW

**Comment by mjgiarlo** [comment]
@ndushay @arcadiafalcone I welcome your thoughts on the WF label language changes in this PR! You can see them in action here: https://argo-qa.stanford.edu/report/workflow_grid

**Review by justinlittman — APPROVED** [review]

**Review by arcadiafalcone — APPROVED** [review]

**Comment by mjgiarlo** [comment]
CircleCI is borked but I ran the builds locally and they pass. Merging.

File: config/workflows/eemsAccessionWF.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   </process>
   <process batch-limit="100" error-limit="5" name="submit-marc">
     <prereq>eems-transfer</prereq>
-    <label>Send stub MARC record to Symphony</label>
+    <label>Send stub MARC record to catalog</label>
```
Comment (ndushay) [inline config/workflows/eemsAccessionWF.xml:16]: EEMS is dead but the workflow lives forever.

**Review by ndushay — COMMENTED** [review]
just catching up on email.  Thanks for this.

**Pull request merged**